v2/interface.go

```diff
@@ -13,14 +13,17 @@ var _ OptionProcessor = (*Goldie)(nil)
 // options to an OptionProcessor.
 type Option func(OptionProcessor) error
 
+type TB testing.TB
```

Comment:
why adding this interface while you could have used testing.TB directly in the Tester interface ?

There is nothing wrong with this, but I don't see the point here.

Reply:
It was to have an interface that could be extended later but i see there is another interface already in the project for that purpose, I have changed to the testing.TB interface.